packages/orbit-components/src/Itinerary/Itinerary.stories.tsx

```diff
@@ -396,14 +396,16 @@ export const Stop = () => {
   return (
     <Stack spacing="large">
       <Heading type="title2">Regular stop</Heading>
-      <ItinerarySegmentStop
-        city={city}
-        station={station}
-        hidden={hidden}
-        date={date}
-        type={type}
-        time={time}
-      />
+      <Itinerary>
```

Comment:
This was causing a bug in Storybook where the minimal width was 0 (because no context was available for this isolated component).